test/test_components.py

```diff
@@ -11,7 +11,7 @@
 import PIL
 
 import gradio as gr
-from gradio.test_data import media_data
+from gradio import media_data
```

Comment:
did you need to apply these changes since you did this in test_data/__init__.py
`
import gradio.media_data as media_data`